QuizApp/Common/Helpers/Primitives+Extensions.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+import Foundation
+
+extension Float {
+    func asCGFloat() -> CGFloat {
+        CGFloat(self)
+    }
+}
+
+extension Int {
+    func asCGFloat() -> CGFloat {
+        CGFloat(self)
+    }
+}
+
+extension Double {
+    func asCGFloat() -> CGFloat {
+        CGFloat(self)
+    }
+}
```

Comment:
We prefer having computed properties over having functions without parameters if there's no action being done (like login, button tap etc)